posthog/models/event/query_event_list.py

```diff
@@ -108,12 +109,14 @@ def query_events_list(
                 ),
                 {"team_id": team.pk, "limit": limit, **condition_params, **prop_filter_params},
                 query_type="events_list",
+                workload=Workload.OFFLINE,
```

Comment:
Non-blocking

I don't think it's that important, but it sounds strange to say this is offline to me. Isn't this user facing i.e. a user is hitting an api endpoint and expecting something back?

I don't think it matters much and is easy to change so 🤷  . Maybe just a comment.

Reply:
Naming things is hard - happy to rename online/offline to something else if you have ideas.

As to why we're singling out events list? It plus historical exports take up ~80% of what's going on on the cluster at any given time and is otherwise problematic. Read more here: https://github.com/PostHog/posthog/issues/13485, https://github.com/PostHog/posthog/pull/13430, https://github.com/PostHog/product-internal/pull/404

Will add a comment to this query though.